applications/search-api/src/main/java/no/dcat/portal/query/DatasetsQueryService.java

```diff
@@ -227,11 +226,17 @@
         BoolQueryBuilder boolQuery = addFilter(theme, publisher, accessRights, search, orgPath, firstHarvested, lastHarvested, lastChanged, subject, provenance, spatial, opendata, catalog);
 
         // set up search query with aggregations
-        SearchRequestBuilder searchBuilder = getClient().prepareSearch("dcat")
-                .setTypes("dataset")
-                .setQuery(boolQuery)
-                .setFrom(from)
-                .setSize(size)
+        SearchRequestBuilder searchBuilder = getClient().prepareSearch("dcat");
+        searchBuilder
```

Comment:
Hvorfor har du endret her ?

Reply:
fordi disse er felles for både title og alle andre.